lab/src/Microsoft.ServiceModel.Syndication/src/SyndicationItemFormatter.cs

```diff
@@ -187,12 +180,12 @@ protected static bool TryParseElement(XmlReader reader, SyndicationPerson person
 
         protected static void WriteAttributeExtensions(XmlWriter writer, SyndicationItem item, string version)
         {
-            SyndicationFeedFormatter.WriteAttributeExtensions(writer, item, version);
+            SyndicationFeedFormatter.WriteAttributeExtensionsAsync(writer, item, version);
         }
 
         protected static void WriteAttributeExtensions(XmlWriter writer, SyndicationCategory category, string version)
         {
```

Comment:
Dangerous method and it will return when WriteAttributeExtensionsAsync is still writing to writer.